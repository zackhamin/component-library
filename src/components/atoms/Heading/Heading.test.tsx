import { Heading } from '.';
import { render, screen } from '@testing-library/react';

describe('Text', () => {
	it('renders correctly', () => {
		const { container } = render(<Heading level={1}>Text</Heading>);
		expect(container).toMatchSnapshot();
	});

	it('renders correct HTML tag based on prop', () => {
		render(<Heading level={2}>Heading</Heading>);
		expect(screen.getByRole('heading', { level: 2 })).toBeInTheDocument();
	});
});
