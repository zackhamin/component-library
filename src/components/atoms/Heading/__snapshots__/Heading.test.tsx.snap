// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
.c0 {
  margin: 0;
  font-family: BrandonGrotesque-Bold,Arial,Helvetica,sans-serif;
  color: #333333;
  font-size: 2.6rem;
  line-height: 140%;
  font-weight: 450;
}

<div>
  <h1
    class="c0"
  >
    Text
  </h1>
</div>
`;
